sharding/interface.go

```diff
@@ -39,6 +39,7 @@ type NodesCoordinator interface {
 
 // PublicKeysSelector allows retrieval of eligible validators public keys
 type PublicKeysSelector interface {
+	GetAllValidatorsPublicKeys() map[uint32][][]byte
```

Comment:
consider sending strings instead of byte arrays because json will convert byte arrays to unwanted formats

Reply:
You right. Good point.